templates/terraform/sweeper_file.go.erb

```diff
@@ -13,6 +13,12 @@ import (
 <%
 sweeper_name = product_ns + object.name
 wrap_path = object&.nested_query&.keys&.first || object.collection_url_key
+listUrlTemplate = object.__product.base_url + object.base_url
+
+listUrlTemplate.sub! "zones/{{zone}}", "aggregated"
```

Comment:
I'm assuming there's basically no non-GCE zonal resources, so we'll only replace GCE resources with this?

Reply:
Yep, other resources usually use `location/` in the base url. I am assuming some consistency in the future, otherwise we will have to use some manual flag.